azure-container-apps/ReadMe.md

```diff
@@ -1,337 +1,349 @@
+4. Paste the Base64 value from your clipboard into the password field, then click **Save**.
+
+## Step 2: Create the container app
+
+1. Open the Azure Portal and go to the [Container Apps](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps) page.
+2. Click **Create**, then fill out the following fields:
+      1. **Resource group**: Choose an existing Resource Group or create a new one.
+      2. **Container app name**: Enter a name you'd like to use, such as `op-scim-con-app`.
+      3. **Region**: Choose the region you prefer.
+      4. **Container App Environment**: Choose an existing one or create a new one with the information in [step 2.1](#21-optional-create-a-container-app-environment).
+3. If you didn't create a new Container App Environment, continue to [step 2.2](#22--container-app-environment).
+
+### 2.1: (Optional) Create a new Container App Environment
+
+If you choose to create a new Container App Environment, give it a name that makes sense for your organization, such as `op-scim-con-app-env`. Then adjust the following:
+
+1. **Environment type**: Choose **Consumption only**.
+2. **Zone Redundancy**: Choose **Disabled**.
+
+After you adjust these options, click **Create**.
+
+### 2.2: Continue creating the container app
+
+1. On the Create Container App page, click **Next : Container >**.
+2. Deselect **Use quickstart image**, then adjust the following:
+	1. **Name**: Enter `op-scim-bridge`.
+	2. **Image source**: Choose **Docker Hub or other registries**.
+	3. **Image and tag:** Enter `1password/scim:v2.9.0`.
+	4. **CPU and Memory**: Choose **0.25 CPU cores, 0.5 Gi memory**, which should be the default option.
+	5. **Environment variables**: Create two with the following details:
+		1. `OP_SESSION` as the Name and  `""` as the Value.
+		2. `OP_REDIS_URL` as the Name and `redis://localhost:6379` as the Value.
+
+3. Click **Next: Bindings >** at the bottom of the page, then click **Next : Ingress >** to continue.
+4. Click **Enabled** to turn on ingress, then adjust the following:
+	1. **Ingress Traffic**: Choose **Accept traffic from anywhere**.
+	2. **Target Port**: Enter `3002`.
+
+5. Click **Review + create** at the bottom of the page, then wait a moment for the page to load and click **Create.**
+
+After the deployment is complete, click **Go to resource**, then continue to step 3.
+
+> [!NOTE]
+> The deployment will fail because it's expecting the second container to deploy successfully. Continue to step 3 to fully deploy the SCIM bridge.
+
+## Step 3: Configure and deploy your SCIM bridge
+
+### 3.1: Create a secret with your `scimsession` file
+
+1. Choose **Secrets** from the Settings section in the sidebar.
+2. Click **Add**, then enter `scimsession` in the Key field.
+3. Paste the base64 value into the **Value** field. This is the `scimsession` item that you added to your account in [step 1](#step-1-configure-the-scimsession-credentials).
```

Comment:
Small wording change to highlight we need the base64 value of the scimsession item from step 1. 
```suggestion
3. Paste the base64 value into the **Value** field. This is the `Base64 value` in your `scimsession` item that you added to your account in [step 1](#step-1-configure-the-scimsession-credentials).
```

Reply:
I'd not use ` `` ` here to surround `Base64 value`, as that isn't "code". Italics / bold make more sense here in my opinion.